resources/assets/components/MultiValueFilter/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { map } from 'lodash';
+import classnames from 'classnames';
+
+import '../Tags/tags.scss';
+
+class MultiValueFilter extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      'good-photo': false,
+      'good-quote': false,
+      'hide-in-gallery': false,
+      'good-for-sponsor': false,
+      'good-for-storytelling': false,
+    }
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(key) {
```

Comment:
this could take in the `key` and `option` so you can just push that to the state.